enhancements/kube-apiserver/enforce-labels.md

```diff
@@ -0,0 +1,198 @@
+---
+title: enforce-labels
+authors:
+ - "@mmirecki"
+reviewers:
+ - "@fedepaol"
+approvers:
+ - TBD
+creation-date: 2020-08-07
+last-updated: 2020-08-07
+status: implementable
+---
+
+# Enforced Labels and Annotations
+
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
```

Comment:
I miss the scope of this. Is it an OLM provided component? Is it supposed to be in every OpenShift installation?

Reply:
Definitely not every OpenShift installation. This feature is required only by some specific customers.

Reply:
Add this as non-goal in the doc.

Reply:
Done